src/components/link/link.scss

```diff
@@ -0,0 +1,97 @@
+@use '../../scss/global.scss';
+
+:host {
+  display: inline-block;
+  margin: 4px 0;
+}
+
+// base class
+.kd-link-text {
+  display: inline-flex;
+  padding: 0 2px;
+  position: relative;
+  text-decoration: none;
+
+  &:hover {
+    ::slotted(kd-icon) {
+      transform: translate(4px, 0);
+    }
+  }
+
+  // Primary (app)
+  &-primary {
+    color: var(--kd-color-text-link);
+
+    &:hover {
+      color: var(--kd-color-text-link-hover);
+      font-weight: var(--kd-font-weight-medium);
```

Comment:
I don't like how changing to bold on hover makes the link jiggle. Let's bring this up to the designers and try to find a better alternative.

Reply:
Yes. yesterday I discussed same with **Suresh G.** But he said for **button : hover** we are making text bold so he did same with link too.

Reply:
Hmm I don't see any bold text on hover in the button specs. Should bring this back up to him.

Reply:
ok.

Reply:
Done.